_posts/2019-12-10-variational-gaussian-process/vgp.Rmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "Gaussian Process Regression with tfprobability"
+description: > 
+ Continuing our tour of applications of TensorFlow Probability (TFP), after Bayesian Neural Networks, Hamiltonian Monte Carlo and State Space Models here we show an example of Gaussian Process Regression. In fact, what we see is a rather "normal" Keras network, defined and trained pretty much the usual way, with TFP's Variational Gaussian Process layer pulling off all the magic. 
+ 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019vgp
+bibliography: bibliography.bib
+date: 12-10-2019
+categories:
+  - Probability and statistics
+  - Keras
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/kernel_cookbook.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+How do you motivate, or come up with a story around, Gaussian Process Regression on a blog primarily dedicated to deep learning? 
+
+Easy. As demonstrated by seemingly-unavoidable, reliably-recurring twitter "wars" surrounding AI, nothing attracts attention like controversy and antagonism. So, let's go back twenty years and find citations of people saying, "here come Gaussian Processes, we don't need to bother with those finicky, hard to tune neural networks anymore!". And today, here we are; everyone knows _something_ about deep learning but who's heard of Gaussian Processes?
+
+While similar tales tell a lot about history of science and and development of opinions, we prefer a different angle here. In the preface to their 2006 book on _Gaussian Processes for Machine Learning_ [@Rasmussen], Rasmussen and Williams say, referring to the "two cultures" -- the disciplines of statistics and machine learning, respectively: ^[In the book, it's "two communities", not "two cultures"; we choose L. Breiman's [@breiman2001] more punchy expression just ... because.]
```

Comment:
```suggestion
While similar tales tell a lot about history of science and development of opinions, we prefer a different angle here. In the preface to their 2006 book on _Gaussian Processes for Machine Learning_ [@Rasmussen], Rasmussen and Williams say, referring to the "two cultures" -- the disciplines of statistics and machine learning, respectively: ^[In the book, it's "two communities", not "two cultures"; we choose L. Breiman's [@breiman2001] more punchy expression just ... because.]
```

Reply:
PS If you're ever oddly curious, I have a _deep_ collection of papers purporting to define two cultures (see Tweet [here](https://twitter.com/dataandme/status/985858420296298497?s=20)). I think that C.P. Snow paper was one of the first I read for an STS course, so it has an eternal place in my heart.

Reply:
oh, super cool! Need to check out some of those...